### PR TITLE
fix(vulnerabilites): upgrade sequelize to v5 + fix other vulnerabilities

### PR DESCRIPTION
**Pull request by NivLipetz**

### CODE REVIEW

File: src/configManager/models/database/sequelize/sequelizeConnector.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ async function deleteConfig(key) {
 async function getConfig() {
     const configClient = client.model('config');
     const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
+        attributes: { exclude: ['updatedAt', 'createdAt'] }
```
Comment (enudler) [inline src/configManager/models/database/sequelize/sequelizeConnector.js]: in sequlize.js in createClient we define

define: {
            underscored: true
        },

not sure why updated_at is now updatedAt
also what about existing users? they have this table already created with underscore name columns

File: src/configManager/models/database/sequelize/sequelizeConnector.js [hunk 2]
```diff
@@ -50,10 +50,10 @@ async function getConfig() {
 async function getConfigValue(configValue) {
     const configClient = client.model('config');
     const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
+        attributes: { exclude: ['updatedAt', 'createdAt'] }
     };
     options.where = { key: configValue };
-    let dbResult = await configClient.find(options);
+    let dbResult = await configClient.findOne(options);
```
Comment (enudler) [inline src/configManager/models/database/sequelize/sequelizeConnector.js]: i would expect that findOne would throw exception if nothing found (also i think it's not the case) 
can you please verify it?
i also would expect it to return a single value and not array as line 57 assumes

File: src/configManager/models/database/sequelize/sequelizeConnector.js (hunk 1 above)
Comment (NivLipetz) [reply src/configManager/models/database/sequelize/sequelizeConnector.js]: You're right, will read about it later here: https://github.com/sequelize/sequelize/issues/6423#issuecomment-379472035
Seems there was a breaking change in underscore

Comment (NivLipetz) [reply src/configManager/models/database/sequelize/sequelizeConnector.js]: /**
   * Override the name of the createdAt column if a string is provided, or disable it if false. Timestamps
   * must be true. Not affected by underscored setting.
   */
  createdAt?: string | boolean;

Comment (NivLipetz) [reply src/configManager/models/database/sequelize/sequelizeConnector.js]: fixed

File: src/configManager/models/database/sequelize/sequelizeConnector.js (hunk 2 above)
Comment (NivLipetz) [reply src/configManager/models/database/sequelize/sequelizeConnector.js]: fixed

**Pull request merged**